CHANGELOG.md

```
@@ -18,11 +27,16 @@
 
 ## [1.2.1](https://github.com/VilnaCRM-Org/website/compare/v1.2.0...v1.2.1) (2025-02-14)
 
-
 ### Bug Fixes
 
-* **#121:** change token rotation logic ([#122](https://github.com/VilnaCRM-Org/website/issues/122)) ([81ba0b0](https://github.com/VilnaCRM-Org/website/commit/81ba0b03684f043dc9c3f090243d32e3f364f6f6)), closes [#121](https://github.com/VilnaCRM-Org/website/issues/121) [#121](https://github.com/VilnaCRM-Org/website/issues/121) [#121](https://github.com/VilnaCRM-Org/website/issues/121) [#121](https://github.com/VilnaCRM-Org/website/issues/121) [#121](https://github.com/VilnaCRM-Org/website/issues/121)
+- **#1:** infinite pnpm install ([#41](https://github.com/VilnaCRM-Org/website/issues/41)) ([23c5c18](https://github.com/VilnaCRM-Org/website/commit/23c5c18f0e7647fe2bd4b82a8c63fb131b69264d)), closes [#1](https://github.com/VilnaCRM-Org/website/issues/1) [#1](https://github.com/VilnaCRM-Org/website/issues/1)
+- **#37:** add changelog.md ([2fd6c57](https://github.com/VilnaCRM-Org/website/commit/2fd6c576a04c26a32520cc10fb15d256bfa9d500)), closes [#37](https://github.com/VilnaCRM-Org/website/issues/37)
+- **#37:** update version in autorelease.yml ([2aa93e0](https://github.com/VilnaCRM-Org/website/commit/2aa93e0b643abc7ed4568097aa29faf37bbbad63)), closes [#37](https://github.com/VilnaCRM-Org/website/issues/37)
 
+### Features
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)